collector/perf_schema_events_statements.go

```diff
@@ -27,9 +27,14 @@ const perfEventsStatementsQuery = `
 	    SUM_SORT_MERGE_PASSES,
 	    SUM_SORT_ROWS,
 	    SUM_NO_INDEX_USED
-	  FROM performance_schema.events_statements_summary_by_digest
-	  WHERE SCHEMA_NAME NOT IN ('mysql', 'performance_schema', 'information_schema')
-	    AND last_seen > DATE_SUB(NOW(), INTERVAL %d SECOND)
+	  FROM (
+	    SELECT *
+	    FROM performance_schema.events_statements_summary_by_digest
+	    WHERE SCHEMA_NAME NOT IN ('mysql', 'performance_schema', 'information_schema')
+	      AND LAST_SEEN > DATE_SUB(NOW(), INTERVAL %d SECOND)
+	    ORDER BY LAST_SEEN DESC
+	  )Q
```

Comment:
Hah, no, it's an implementation detail of having to assign the sub-query to a virtual table.